src/controllers/admin/scan/scan.ts

```diff
@@ -2,11 +2,12 @@ import Joi from 'joi';
 import { NextFunction, Request, Response } from 'express';
 import { hasPermission } from '../../../middlewares/authentication';
 import { Permission, Error } from '../../../types';
-import { badRequest, forbidden, noContent, notFound } from '../../../utils/responses';
+import { badRequest, forbidden, noContent, notFound, success } from '../../../utils/responses';
```

Comment:
```suggestion
import { badRequest, forbidden, notFound, success } from '../../../utils/responses';
```
Pour que les tests (et eslint) fonctionnent 😉